src/BBCodeParser.php

```diff
@@ -12,143 +12,146 @@
+                    "Description" => _t('BBCodeParser.UNORDEREDDESCRIPTION', 'Unordered list'),
+                    "Example" => '[ulist][*]'._t('BBCodeParser.UNORDEREDEXAMPLE1', 'unordered item 1').'[/ulist]'
+                )),
+                new ArrayData(array(
+                    "Title" => _t('BBCodeParser.IMAGE', 'Image'),
+                    "Description" => _t('BBCodeParser.IMAGEDESCRIPTION', 'Show an image in your post'),
+                    "Example" => "[img]http://www.website.com/image.jpg[/img]"
+                )),
+                new ArrayData(array(
+                    "Title" => _t('BBCodeParser.LINK', 'Website link'),
+                    "Description" => _t('BBCodeParser.LINKDESCRIPTION', 'Link to another website or URL'),
+                    "Example" => '[url]http://www.website.com/[/url]'
+                )),
+                new ArrayData(array(
+                    "Title" => _t('BBCodeParser.LINK', 'Website link'),
+                    "Description" => _t('BBCodeParser.LINKDESCRIPTION', 'Link to another website or URL'),
+                    "Example" => "[url=http://www.website.com/]Website[/url]"
+                ))
+            )
+        );
+    }
 
-	public function useable_tagsHTML() {
-		$useabletags = "<ul class='bbcodeExamples'>";
-		foreach($this->usable_tags()->toArray() as $tag){
-			$useabletags = $useabletags."<li><span>".$tag->Example."</span></li>";
-		}
-		return $useabletags."</ul>";
-	}
+    public function useable_tagsHTML()
+    {
+        $useabletags = "<ul class='bbcodeExamples'>";
+        foreach ($this->usable_tags()->toArray() as $tag) {
+            $useabletags = $useabletags."<li><span>".$tag->Example."</span></li>";
+        }
+        return $useabletags."</ul>";
+    }
 
-	/**
-	 * Main BBCode parser method. This takes plain jane content and
-	 * runs it through so many filters
-	 *
-	 * @return DBField
-	 */
-	public function parse() {
-		// Convert content to plain text
-		/** @var DBHTMLText $fragment */
-		$fragment = DBField::create_field('HTMLFragment', $this->content);
-		$this->content = $fragment->Plain();
+    /**
+     * Main BBCode parser method. This takes plain jane content and
+     * runs it through so many filters
+     *
+     * @return DBField
+     */
+    public function parse()
+    {
+        // Convert content to plain text
+        /** @var DBHTMLText $fragment */
+        $fragment = DBField::create_field('HTMLFragment', $this->content);
+        $this->content = $fragment->Plain();
 
-		// Get options
-		$config = parse_ini_file('BBCodeParser.ini', true);
-		$options = $config['SSHTMLBBCodeParser'];
+        // Get options
+        $config = parse_ini_file('BBCodeParser.ini', true);
+        $options = $config['SSHTMLBBCodeParser'];
 
-		require_once(__DIR__ . '/../thirdparty/HTML_BBCodeParser2/HTML/BBCodeParser2.php');
+        require_once BBCODEPARSER_PATH . '/thirdparty/HTML_BBCodeParser2/HTML/BBCodeParser2.php';
```

Comment:
can we use composer to add this dir to the include path which would save us having to use this constant?

see framework for an example: https://github.com/silverstripe/silverstripe-framework/blob/master/composer.json#L69-L72